plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/GlueHiveMetastore.java

```diff
@@ -184,6 +179,7 @@
     public GlueHiveMetastore(
             HdfsEnvironment hdfsEnvironment,
             GlueHiveMetastoreConfig glueConfig,
+            @ForGlueHiveMetastore AWSCredentialsProvider credentialsProvider,
```

Comment:
you may want to add a `requireNonNull` check for the new constructor argument`

Reply:
cc: @findinpath